src/components/LogModal.tsx

```diff
@@ -78,12 +77,13 @@ const LogModal = ({ instance, fileHeader }: ILogModal) => {
         logID.current = response.headers.request_id
       })
       .catch((e) => {
```

Comment:
`e` is an `AxiosError` where `response` is optional. So for maximum safety `e.response.data.message ||` should probably be something like `e.response?.data.message ||`

I believe that if you include type information for the error as `e: AxiosError`, VS code will warn you about this.